fig.linux.yml

```diff
@@ -0,0 +1,37 @@
+db:
+  build: db
+  ports:
+    - 5432:5432
+  environment:
+    - HATCH_DB_USER
+    - HATCH_DB_PASSWORD
+    - HATCH_DB_NAME
+
+server:
+  build: server
+  command: python runserver.py -h 0.0.0.0 -p 5000
+  volumes:
+    - server/:/code
```

Comment:
Since this is essentially the same as `$PWD/server/:/code`, may not need to have two separate `fig.ymls`